### DISCUSSION TITLE
Dead time defaut setting !

### DISCUSSION THREAD

**Discussion by navr32** [dbuezas/arduino-web-timers#6] [General]
After some test i think this will be good always init the dead time to 
about 32..not many more.. .. or not many less to eases the view on oscilloscpe for some beginner testing with it..because if init to 128 for some timer setting this have the effect to full stop the output of the opposite output so someone
which don't know well the working of timers may thinking the settings are bads ...but it is goods..

**Comment by dbuezas** [comment]
Good Idea! This gets a bit more complicated when a different value for TOP is used, but I can initialize it dynamically to half of the otherwise active time of the corresponding output. I'll need to play a bit with it myself.

**Comment by dbuezas** [comment]
I finally got back to the project, so I implemented your suggestion. They now default to 32. Thank you!

**Comment by dbuezas** [comment]
Oh, well, that didn't go that well: Timer0 also have dead time support, but it is an 8 bit timer. Because of that, the max values for the deadime bits is just 2^4. I changed the defaults to be `counterMax^0.3`, this way the initial value is always within range
